support-frontend/conf/PROD.public.conf

```
@@ -19,7 +19,7 @@ metric.url="https://metric-push-api-prod.support.guardianapis.com/metric-push-ap
 play.filters.csp.directives {
   # both https://www.theguardian.com/ https://theguardian.com/ needed for the gu.com/font-loader endpoint to work
   default-src="'self' https://ccpa-pm.sp-prod.net https://cdn.privacy-mgmt.com https://gdpr-tcfv2.sp-prod.net https://ccpa-service.sp-prod.net https://ccpa-notice.sp-prod.net https://sourcepoint.theguardian.com https://ccpa.sp-prod.net https://services.postcodeanywhere.co.uk https://stripe-intent.support.guardianapis.com members-data-api.theguardian.com metric-push-api-prod.support.guardianapis.com www.paypalobjects.com www.paypal.com t.paypal.com www.sandbox.paypal.com js.stripe.com https://payments.amazon.com https://payments-sandbox.amazon.com https://coin.amazonpay.com https://api-cdn.amazon.com static-na.payments-amazon.com ophan.theguardian.com j.ophan.co.uk media.guim.co.uk uploads.guim.co.uk www.google-analytics.com www.googletagmanager.com tagmanager.google.com assets.guim.co.uk www.googleadservices.com googleads.g.doubleclick.net www.google.com www.google.co.uk static.ads-twitter.com bat.bing.com bid.g.doubleclick.net t.co analytics.twitter.com stats.g.doubleclick.net www.youtube-nocookie.com connect.facebook.net www.facebook.com checkout.stripe.com fonts.googleapis.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com sentry.io *.quantummetric.com blob: data: wss: 'unsafe-inline' q.stripe.com payment.guardianapis.com https://interactive.guim.co.uk/ https://www.theguardian.com/ https://theguardian.com/ https://contribution-referrals.support.guardianapis.com/referral-code"
-  frame-ancestors="https://gnmtouchpoint--c.eu31.visual.force.com https://gnmtouchpoint.lightning.force.com https://www.theguardian.com"
+  frame-ancestors="https://gnmtouchpoint--c.eu31.visual.force.com https://gnmtouchpoint.lightning.force.com https://www.theguardian.com https://gnmtouchpoint--dev1--c.vf.force.com"
```

```suggestion
  frame-ancestors="https://gnmtouchpoint--c.eu31.visual.force.com https://gnmtouchpoint.lightning.force.com https://www.theguardian.com https://gnmtouchpoint--c.vf.force.com"
```

Sorry, that's my typo!